test/Parse/trailing-comma.swift

```diff
@@ -0,0 +1,139 @@
+// RUN: %target-swift-frontend -parse -verify %s -disable-experimental-parser-round-trip
```

Comment:
Why do we need to add `-disable-experimental-parser-round-trip`? If we do, that indicates that there is a round-trip bug in the new parser that should be fixed.

Reply:
Honestly I just copied this from another test file 😅 I will remove it!